ext/standard/tests/mail/mail_variation3.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+Test mail() function : variation sendmail temp fail
+--INI--
+sendmail_path=exit 75
+--SKIPIF--
+<?php
+if(substr(PHP_OS, 0, 3) == "WIN")
+  die("skip Won't run on Windows");
+?>
+--FILE--
+<?php
+echo "*** Testing mail() : variation ***\n";
+
+// Initialise all required variables
+$to = 'user@example.com';
+$subject = 'Test Subject';
+$message = 'A Message';
+var_dump( mail($to, $subject, $message) );
+?>
+--EXPECT--
+*** Testing mail() : variation ***
+bool(true)
```

Comment:
Is this expectation correct?

Reply:
yes, since `EX_TEMPFAIL` is defined as 75,

```
if ((ret != EX_OK)&&(ret != EX_TEMPFAIL))
```

Reply:
I'm not sure this is useful, since the mail still go into the void without the script noticing, but this certainly a different issue. Thanks for clarifying.